component/src/main/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/AbstractRegressor.java

```diff
@@ -0,0 +1,34 @@
+package org.wso2.extension.siddhi.gpl.execution.streamingml.regression;
```

Comment:
License is missing

Reply:
Updated license to GPL in [#670e8f4657dbbf5dfb89bcb093bf4922d59b48d5](https://github.com/pcnfernando/siddhi-gpl-execution-streamingml/commit/670e8f4657dbbf5dfb89bcb093bf4922d59b48d5)